scripts/generate_datasets.sh

```diff
@@ -0,0 +1,15 @@
+# Script to copy generated dataset pages to the local docsite repo.
```

Comment:
I wonder if this script should be in g3?  Given its reference to blaze run etc.

For example, the script gets run from a g4 client, takes the github docsite dir as argument, does the blaze run to produce stuff in /tmp/dataset_info, copies over the *.md to github dir, prepares a commit with the modified files, and informs user to review commit and send out PR (like https://github.com/datacommonsorg/mixer/blob/master/scripts/prepare_release.sh#L60-L62).

Reply:
sounds good, removed and will add a script to g3